### ISSUE TITLE
overlaysWebView default

### ISSUE THREAD

**Issue by leecrossley** [phonegap-build/StatusBarPlugin#5]
Overlaying the WebView is the default behaviour without using this plugin. When using the plugin you have to forcibly call `StatusBar.overlaysWebView(true);` e.g.

```
StatusBar.styleLightContent();
StatusBar.overlaysWebView(true); // required
```

The 2nd line shouldn't be necessary in my opinion. Thoughts?

**Comment by bobeasterday** [comment]
When we initially added support on Build for Cordova 3.1.0, we got substantial pushback from the community because it defaulted to iOS7 behavior (status bar overlays the web view)

We opted to make 3.1.0 default to iOS6 behavior (on PhoneGap Build) and allow the developer to use the plugin to set it back to iOS7 behavior at their discretion.

Doing the reverse, would have required users to use the plugin (as opposed to optionally) or to pad their content to compensate.

Its a judgement call either way which we will revisit when we add support for 3.2.0.
